src/components/vms/hostvmslist.jsx

```diff
@@ -63,9 +64,23 @@ const _ = cockpit.gettext;
  * List of all VMs defined on this host
  */
 const HostVmsList = ({ vms, config, ui, storagePools, actions, networks, onAddErrorNotification }) => {
+    const [virtualizationEnabled, setVirtualizationEnabled] = useState(true);
     const [statusSelected, setStatusSelected] = useState({ value: _("All"), toString: function() { return this.value } });
     const [currentTextFilter, setCurrentTextFilter] = useState("");
     const [statusIsExpanded, setStatusIsExpanded] = useState(false);
+
+    useEffect(() => {
+        async function checkVirtualization() {
+            try {
+                await cockpit.script("grep -E 'vmx|svm|0xc0f' --color=auto /proc/cpuinfo");
```

Comment:
`grep -E "vmx|svm" /proc/cpuinfo` is enough. Color just highlights the text when users read it.

However this will not trow an exception, if will just return an empty ouput. 

So you need something like: 
````
cockpit.script("grep -E 'vmx|svm'  /proc/cpuinfo").then(output => setVirtualiationEnabled(!!output.trim()))
```

Reply:
Thanks for looking into it.

It throws an error at least in my case.

Also in grep manual (`man grep`):

![image](https://user-images.githubusercontent.com/108616679/225239780-92c9654e-fc1c-4b72-85b0-f548b21ed010.png)

I think it's better to not rely on exit codes or whether `grep` return an empty string. It might not even be consistent across systems.

Using `await cockpit.file("/proc/cpuinfo").read()` instead, suggested by @martinpitt, seems better to me.

What do you think?

Reply:
martins suggestion looks good!